Documentation/Elastics/ElasticSystem.md

```diff
@@ -0,0 +1,76 @@
+# Elastic system (experimental)
+
+![Elastic System](../Images/Elastics/Elastics_Main1.gif)
+
+MRTK comes with an elastic simulation system that includes a wide variety of extensible and flexible subclasses, offering bindings for 4-dimensional quaternion springs, 3-dimensional volume springs and simple linear spring systems.
+
+Currently the following MRTK components supporting the [elastics manager](xref:Microsoft.MixedReality.Toolkit.Experimental.Physics.ElasticsManager) can leverage elastics functionality:
+
+- [Bounds control](../README_BoundsControl.md)
+- [Object manipulator](../README_ObjectManipulator.md)
+
+## Elastics manager
+![Elastic System2](../Images/Elastics/Elastics_Main.gif)
+
+The elastics manager takes over processing passed transforms and feeding them into the elastics system. 
+
+Enabling elastics for custom components can be achieved by two steps:
+1. Calling the Initialize method on manipulation start, updating the system with the current host transform.
+1. Querying ApplyHostTransform whenever a elastics calculation should be performed on the updated target transform.
+
+Note that elastics will continue simulating once manipulation ends (through the elastics manager update loop). To block elastics auto update [EnableElasticsUpdate](xref:Microsoft.MixedReality.Toolkit.Experimental.Physics.ElasticsManager.EnableElasticsUpdate) can be set to false.
```

Comment:
```suggestion
Note that elastics will continue simulating once manipulation ends (through the elastics manager update loop). To block the behavior, elastics auto update [EnableElasticsUpdate](xref:Microsoft.MixedReality.Toolkit.Experimental.Physics.ElasticsManager.EnableElasticsUpdate) can be set to false.
```

Reply:
I think the auto update text here is clearer (does behavior in the suggestion apply to the update loop? manipulation end? continuing simulating?) Though this is pretty minor as far as clarity goes